src/shared/getTokenIcon.tsx

```diff
@@ -0,0 +1,45 @@
+import BTC from "@components/icons/BTC";
+import dBCH from "@components/icons/dBCH";
+import DFI from "@components/icons/DFI";
+import dBTC from "@components/icons/dBTC";
+import dDFI from "@components/icons/dDFI";
+import dETH from "@components/icons/dETH";
+import dLTC from "@components/icons/dLTC";
+import dUSDT from "@components/icons/dUSDT";
+import dUSDC from "@components/icons/dUSDC";
+import dUSD from "@components/icons/dUSD";
+import DefaultToken from "@components/icons/DefaultToken";
+
+interface MappingProps {
+  size?: number;
+  symbol?: string;
+}
+
+const mapping: Record<string, ({ size, symbol }: MappingProps) => JSX.Element> =
+  {
+    _UTXO: DFI,
+    DFI,
+    "DFI (UTXO)": DFI,
+    "DFI (Token)": DFI,
+    BTC,
+    dBCH,
+    dBTC,
+    dDFI,
```

Comment:
the white background dfi icon is obsolete
```suggestion
    dDFI: DFI,
```

Reply:
updated